lib/roo/excelx.rb

```diff
@@ -60,15 +60,18 @@ def initialize(filename_or_stream, options = {})
       @filename = local_filename(filename_or_stream, @tmpdir, packed)
       process_zipfile(@filename || filename_or_stream)
 
-      @sheet_names = workbook.sheets.map do |sheet|
+      sheet_indicies = []
+      @sheet_names = workbook.sheets.each_with_index.map do |sheet, index|
```

Comment:
Why not use `select` instead of `map` if you have a condition inside?

Reply:
@simonoff The condition was already there before the change, so I left that as is, but also `select` would return sheets instead of sheet names.